Include/internal/pycore_gil.h

```diff
@@ -25,7 +25,7 @@ struct _gil_runtime_state {
     unsigned long interval;
     /* Last PyThreadState holding / having held the GIL. This helps us
        know whether anyone else was scheduled after we dropped the GIL. */
-    _Py_atomic_address last_holder;
+    PyThreadState* last_holder;
     /* Whether the GIL is already taken (-1 if uninitialized). This is
        atomic because it can be read without any lock taken in ceval.c. */
     _Py_atomic_int locked;
```

Comment:
@vstinner @colesbury 

https://github.com/python/cpython/blob/982f1b7d6dc2f13b9607ce092e36e32972e3702c/Python/ceval_gil.c#L179
https://github.com/python/cpython/blob/982f1b7d6dc2f13b9607ce092e36e32972e3702c/Python/ceval_gil.c#L194

IIUC, To update `locked` field we need to introduce `_Py_atomic_load/store_int_release/acquire` 

Do you have better ideas?

Reply:
It sounds like a good idea, for another PEP